cmd/nerdctl/run.go

```diff
@@ -511,6 +515,12 @@ func runAction(clicontext *cli.Context) error {
 		}
 	}
 
+	ulimitOpts, err := generateUlimitsOpts(clicontext)
```

Comment:
I think it is better to check `strutil.DedupeStrSlice(clicontext.StringSlice("ulimit"))` len here